Source/Streamstone/Stream.Operations.cs

```diff
@@ -31,7 +31,7 @@ public Stream Execute()
 
                 try
                 {
-                    table.Execute(insert.Prepare());
+                    table.ExecuteAsync(insert.Prepare()).Wait();
```

Comment:
These are deadlocks waiting to happen, right? Sync over async is as bad as it gets, especially in a library.